hack/scripts/provision.sh

```diff
@@ -321,6 +323,7 @@ containerd-socket: "$CONTAINERD_STATE_DIR/containerd.sock"
 grpc-endpoint: "$address:9090"
 verbosity: 9
 parent-iface: "$parent_iface"
+insecure: $insecure
```

Comment:
Any checks if `$insecure` contains `1|0` or `true|false`, or in that case we just say "user error, Read The Fancy Manual"

Reply:
I am doing more around this soon, so this will likely change, just need to get unblocked on something.

We don't validate other inputs (for this users cannot actually give any input since it is a bool flag) so perhaps that is a ticket for another day